### PR TITLE
DIABLO-36, complete dir structure with tests, configs, .travis.yml and buildspec

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/DIABLO-36
https://jira.ets.berkeley.edu/jira/browse/DIABLO-37

BOA devs will find this dir structure and naming convention quite familiar. With this PR, I'm also testing Travis and Slack integration.

### CODE REVIEW

**Comment by ets-readonly** [comment]
`psycopg2` needed in requirements. Amending.

File: README.md [hunk 1]
```diff
@@ -1 +1,67 @@
-# diablo
\ No newline at end of file
+# Diablo
+
+UC Berkeley at Course Capture
```
Comment (johncrossman) [inline README.md]: This typo will be fixed in the next PR.

Comment (paulkerschen) [reply README.md]: UCC Berkeley at California. UBCalifornia at Capture. UniCourseBCaptured at USB-C.

**Comment by paulkerschen** [comment]
I feel like I've seen this somewhere before.

File: README.md (hunk 1 above)
Comment (paulkerschen) [reply README.md]: This is a Diablo comment.

**Pull request merged**